pkg/platform/runtime/buildscript/buildscript_test.go

```diff
@@ -217,15 +240,16 @@ func TestExample(t *testing.T) {
 			},
 		},
 		&In{Name: ptr.To("runtime")},
+		expr,
 	}, script)
 }
 
 func TestString(t *testing.T) {
 	script, err := NewScript([]byte(
 		`let:
     runtime = solve(
-        requirements=[{name="language/python"}],
-        platforms=["12345", "67890"]
+        platforms=["12345", "67890"],
+        requirements=[{name="language/python"}]
```

Comment:
Was this order messing with the tests? If so then we need to allow for arbitrary order of solve arguments.

Reply:
This is why I filed DX-2221 (Implement a canonical/stable sort for buildscript output.) It doesn't have anything to do with build expressions, just with the order that the build scripts are written in.